noir-projects/noir-protocol-circuits/crates/types/src/abis/public_call_stack_item_compressed.nr

```diff
@@ -0,0 +1,75 @@
+use crate::abis::{call_context::CallContext, function_data::FunctionData};
+use crate::address::AztecAddress;
+use crate::constants::{GENERATOR_INDEX__CALL_STACK_ITEM, PUBLIC_CALL_STACK_ITEM_COMPRESSED_LENGTH};
+use crate::traits::{Hash, Empty, Serialize, Deserialize};
+use crate::utils::reader::Reader;
+
+struct PublicCallStackItemCompressed {
```

Comment:
Could you please add here a comment describing wtf is this and why we need it?

Reply:
Will add 🫡